google-cloud-core/pom.xml

```diff
@@ -101,22 +101,22 @@
     <dependency>
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java</artifactId>
-      <version>3.0.0-beta-3</version>
+      <version>3.0.0</version>
```

Comment:
I would prefer to just keep the stack consistent; grpc depends on 3.0.0, and all of the grpc-generated packages reference 3.0.0. I kind of want to get this in and move on.